docs/content/tutorials/create-environment/index.md

```diff
@@ -0,0 +1,169 @@
+---
+type: docs
+title: "4. Create Environment"
+linkTitle: "4. Create Environment"
+description: "Create a Radius Environment to deploy the application"
+weight: 400
+---
+
+In part four of this tutorial, you will create an environment to register the Recipe for the PostgreSQL resource type.
+
+## Create a workspace
+
+Radius Workspace is a combination of a Kubernetes context, a Radius Environment, and a Resource Group.
```

Comment:
Add links to the concept pages for Workspaces, Environments, and Resource Groups. i.e.
https://docs.radapp.io/guides/operations/workspaces/overview/
https://docs.radapp.io/guides/deploy-apps/environments/overview/
https://docs.radapp.io/guides/operations/groups/overview/

```suggestion
A Radius Workspace is a combination of a Kubernetes context, a Radius Environment, and a Radius Resource Group.
```